src/Model/PhasorDynamics/SystemModel.hpp

```diff
@@ -144,6 +170,19 @@ namespace GridKit
           addComponent(gen);
         }
 
+        // Add governors
+        for (const auto& govdata : data.gov)
+        {
+
+          /* Need to add signals to gendata class */
+          auto* gov = new Governor::Tgov1<ScalarT, IdxT>(
+              getBus(govdata.signal_pmech),
+              getBus(govdata.signal_speed),
+              govdata);
```

Comment:
is it the case with `Tgov1` that these signals are non-optional? (that sounds correct to me but i want to make sure as it is not documented in the input format)